### PR TITLE
Fix #5837  user profile pic should be displayed wherever we are showing the username

### LINKED ISSUE #5837: user profile pic should be displayed wherever we are showing the username
**Feature**
User profile pic should be displayed whenever we are showing the username. In Select box in search page, entity page, task pages

**Describe the task**
Do a entire sweep of the UI to make sure we display the user profile pic

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Closes #5837 

### Describe your changes :

I worked on issue #5837 

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/Assignees.tsx [hunk 1]
```diff
@@ -35,6 +38,26 @@ const Assignees: FC<Props> = ({ assignees, onSearch, onChange, options }) => {
...
+      <Tag className="assignee-tag" closable={closable} onClose={onClose}>
+        <Space>
+          <ProfilePicture
+            id={value}
+            name={label as unknown as string}
+            width="22"
+          />
+          <span>{label}</span>
+        </Space>
+      </Tag>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/Assignees.tsx:57]: Can we try this here: https://ant.design/components/tag/#components-tag-demo-icon

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/Assignees.tsx [hunk 2]
```diff
@@ -35,6 +38,26 @@ const Assignees: FC<Props> = ({ assignees, onSearch, onChange, options }) => {
...
+    return (
+      <Tag className="assignee-tag" closable={closable} onClose={onClose}>
+        <Space>
+          <ProfilePicture
+            id={value}
+            name={label as unknown as string}
+            width="22"
+          />
+          <span>{label}</span>
+        </Space>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/Assignees.tsx:56]: Let's create it as component and use it across like: `<UserTag>`

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/Assignees.tsx [hunk 3]
```diff
@@ -55,7 +79,10 @@ const Assignees: FC<Props> = ({ assignees, onSearch, onChange, options }) => {
           data-testid="assignee-option"
           data-usertype={option.type}
           key={option.value}>
-          {option.label}
+          <Space>
+            <ProfilePicture id={option.value} name={option.label} width="22" />
+            <span>{option.label}</span>
+          </Space>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/Assignees.tsx:85]: `<UserTag>`

File: openmetadata-ui/src/main/resources/ui/src/components/dropdown/DropDownList.tsx [hunk 4]
```diff
@@ -117,14 +119,20 @@ const DropDownList: FunctionComponent<DropDownListProp> = ({
...
+              name={item.name as string}
+              width="22"
+            />
+          )}
+          <p
+            className={classNames('tw-truncate', widthClass)}
+            title={item.name as string}>
+            {item.name}
+          </p>
+        </Space>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/dropdown/DropDownList.tsx:135]: We can use UserTag here right?

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/dropdown/DropDownList.tsx:135]: Done.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**